odk2-src/advanced-topics-developer.rst

```diff
@@ -0,0 +1,11 @@
+Platform Developer Advanced Topics
+=======================================
+
```

Comment:
I would repeat your definition of a Platform Developer since a reader may not have read the intro